### ISSUE TITLE
The cheatsheet popup's show too fast.

### ISSUE THREAD

**Issue by fourq** [cljsinfo/cljs.info#65]
The cheatsheet popups are pretty intrusive since they show up so fast and cover so much of the screen. If would be nice to add a short delay as to allow the mouse pointer to find what it's looking for while hovering around. 

Here's what I mean: http://screencast.com/t/3pTfLyi2RAHR

**Comment by shaunlebron** [comment]
yeah, we've both complained about this.  It's a serious UX issue.  I think the user can naturally discover the presence of the tooltip without showing it immediately on hover.  I think that immediacy makes it too sticky.  thanks for filing the issue.

**Comment by comamitc** [comment]
IMO, slowing down the animation and adding #20 will improve UX. You are probably looking for something specific when on the cheatsheet. If you are looking for generic documention to "explore" in #69 accomplishes this in much the same way as the other heavy hitting programming languages. 

http://docs.oracle.com/javase/7/docs/api/
http://www.scala-lang.org/api/current/#package
